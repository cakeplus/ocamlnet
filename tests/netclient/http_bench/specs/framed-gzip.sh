. helpers.sh

start_test_server -line 1 -file data/framed-gzip
trap "stop_test_server" EXIT
request -gzip -response 'ABCDEFGHIJKLMNOPQRSTUVWXYZ: abcdefghijklmnopqrstuvwxyz
' -get / -run
