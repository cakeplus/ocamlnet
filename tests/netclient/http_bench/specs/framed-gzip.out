C97F41A17B977A91D884D197F5D97F89
